torch/csrc/api/src/nn/modules/loss.cpp

```diff
@@ -1,5 +1,8 @@
 #include <torch/nn/modules/loss.h>
-
+<<<<<<< HEAD
+=======
+using namespace std
+>>>>>>> Add C++ Frontend - triplet-margin-loss
```

Comment:
We need to fix the code conflict here, and we also shouldn't do `using namespace std`.